lib/dom.js

```diff
@@ -158,6 +160,7 @@ type DragEventTypes = 'drag' | 'dragend' | 'dragenter' | 'dragexit' | 'dragleave
 type PointerEventTypes = 'pointerover' | 'pointerenter' | 'pointerdown' | 'pointermove' | 'pointerup' | 'pointercancel' | 'pointerout' | 'pointerleave' | 'gotpointercapture' | 'lostpointercapture';
 type AnimationEventTypes = 'animationstart' | 'animationend' | 'animationiteration';
 type ClipboardEventTypes = 'clipboardchange' | 'cut' | 'copy' | 'paste';
+type MessageEventTypes = string;
```

Comment:
Is there a reason this is just `string` and not a union of string literals like the rest? Are the MessageEvents not defined here: https://developer.mozilla.org/en-US/docs/Web/Events ?

Reply:
Browser is allowed to listen to specific events choosed by developer :

```JavaScript
evtSource.addEventListener("ping", function(e) {
  var newElement = document.createElement("li");
  
  var obj = JSON.parse(e.data);
  newElement.innerHTML = "ping at " + obj.time;
  eventList.appendChild(newElement);
}, false);
```

https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events

Reply:
Thanks!